lib/hacks/grid-area.js

```diff
@@ -0,0 +1,72 @@
+const parser = require('postcss-value-parser');
+
+const Declaration = require('../declaration');
+const translateShorthand = require('./grid-shorthand');
+
+class GridArea extends Declaration {
+
+    static names = [
+        'grid-area'
+    ];
+
+    /**
+     * Check if -ms-grid-row is already present
+     */
+    check(decl) {
+        for (const i in decl.parent.nodes) {
+            if (decl.parent.nodes.hasOwnProperty(i)) {
+                const element = decl.parent.nodes[i];
+                if (element.prop === '-ms-grid-row') {
+                    return false;
+                }
+            }
+        }
+        return true;
+    }
+
+    /**
+     * Translate grid-area to separate -ms- prefixed properties
+     */
+    insert(decl, prefix) {
+        if (prefix !== '-ms-') {
+            return;
+        }
+
+        const node = parser(decl.value);
+
+        let values = [];
+        let current = 0;
+        values[current] = [];
+
+        for (const i of node.nodes) {
+            if (i.type === 'div') {
+                current += 1;
+                values[current] = [];
+            } else if (i.type === 'word') {
+                values[current].push(i.value);
+            }
+        }
+
+        const [rowStart, rowSpan] = translateShorthand(values, 0, 2);
+        const [columnStart, columnSpan] = translateShorthand(values, 1, 3);
+
+        rowStart && decl.cloneBefore({
```

Comment:
`if (rowStart)` is more simple construction. It will be better to future developers.